### PR TITLE
[ Wait for #1952 ][ Layers ] paralleize Forwarding of conv2d  @open sesame 07/19 16:13

### PR DESCRIPTION
**Pull request by jijoongmoon**
This patch includes the batch direction parallelization of forwarding
in Conv2D layer.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: nntrainer/layers/conv2d_layer.cpp [hunk 1]
```diff
@@ -397,6 +399,8 @@ void Conv2DLayer::forwarding(RunLayerContext &context, bool training) {
   filter_kernel.reshape(filter_dim_squeezed);
 
   Tensor &im2col_result = context.getTensor(wt_idx[ConvParams::inter_result]);
+  im2col_result.setZero();
```
Comment (songgot) [inline nntrainer/layers/conv2d_layer.cpp:402]: I wonder if 'im2col_result.setZero()' has association with comment on line from 404 to 407.
If so, should it be moved?

Comment (jijoongmoon) [reply nntrainer/layers/conv2d_layer.cpp]: right. fixed. Thanks.

**Review by songgot — APPROVED** [review]
LGTM

File: nntrainer/utils/nntr_threads.h [hunk 2]
```diff
@@ -34,6 +34,9 @@ class ParallelBatch {
 public:
   /**
    * @brief Construct a new ParallelBatch object
+   * @param Threadedcb the function run in thread
```
Comment (lhs8928) [inline nntrainer/utils/nntr_threads.h:37]: please add underscore to Threadedcb

**Review by lhs8928 — APPROVED** [review]
LGTM

File: nntrainer/utils/nntr_threads.h (hunk 2 above)
Comment (jijoongmoon) [reply nntrainer/utils/nntr_threads.h]: done. Thanks.

**Pull request merged**